src/test/java/com/salesforce/dynamodbv2/testsupport/ArgumentBuilder.java

```diff
@@ -66,13 +67,13 @@ public ArgumentBuilder(AmazonDynamoDB rootAmazonDynamoDb) {
 
     @Override
     public List<TestArgument> get() {
-        return getAmazonDynamoDbStrategies().stream().flatMap(
-            (Function<AmazonDynamoDB, Stream<TestArgument>>) amazonDynamoDB ->
-                getHashKeyAttrTypes().stream().flatMap(
-                    (Function<ScalarAttributeType, Stream<TestArgument>>) scalarAttributeType ->
-                        Stream.of(new TestArgument(amazonDynamoDB,
-                            getOrgs(),
-                            scalarAttributeType)))).collect(Collectors.toList());
+        List<TestArgument> ret = Lists.newArrayList();
```

Comment:
no functional change here, was tinkering with this to add managed tables to TestArgument. Backtracked that approach, to instead make isMtTable() public, but left this refactoring in, just since I find the flatMap stream approach much less readable given what the streams are doing.

Reply:
:thumbsup: